cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -320,6 +321,62 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     clusterDashboard.waitForPage(undefined, 'cluster-events');
   });
 
+  it('can navigate to Cluster Management Page', () => {
```

Comment:
as this is the root navigation to the area could this `it` come first? that way if it fails it's clear all other failures will be due to a navigation issue